docs/devices/cloud-protocol/pgps.schema.json

```
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/blob/saga/docs/devices/cloud-protocol/pgps.schema.json",
+    "title": "Asset Tracker v2 P-GPS request",
+    "description": "Describes the format which is used by the device to request P-GPS data. ",
+    "type": "object",
+    "properties": {
+        "count": {
+            "type": "integer",
+            "description": "The number of predictions requested. Default is 42 (7 days, 6 predictions per day)",
+            "examples": [42]
+        },
+        "interval": {
+            "type": "integer",
+            "description": "The time between predictions (in minutes). Defaults to 240 (4 hours). Valid values are: 120,240,360,480",
+            "minimum": 120,
+            "maximum": 480,
+            "examples": [120, 240, 360, 480]
+        },
+        "day": {
+            "type": "integer",
+            "description": "The start day of the prediction set as GPS Day (days since the GPS epoch). Defaults to current GPS day.",
+            "minimum": 0,
+            "examples": [15160]
+        },
+        "time": {
+            "type": "integer",
+            "description": "The start time of the prediction set as seconds in day. Defaults to 0. Valid range 0..86399",
```

```suggestion
            "description": "The start time of the prediction set as seconds in day. Defaults to 0.",
```

Valid range is defined in the schema.